extensions/cornerstone/src/utils/measurementServiceMappings/EllipticalRoi.js

```diff
@@ -33,7 +33,9 @@ const EllipticalRoi = {
       SeriesInstanceUID
     );
 
-    const { start, end } = measurementData.handles;
+    const { cachedStats, handles } = measurementData;
```

Comment:
It's silly, but I don't know that we've fixed the "cachedStats" are sometimes out of date issue (based on timing for event and throttled caching). If it's fixed, great. If not, we can fix, or calculate from the handles.